Traceback (most recent call last):
  File "/home/shz19039/sasp/GAE_train.py", line 183, in <module>
    main(seed = args.seed, alpha = alpha, device = device)
  File "/home/shz19039/sasp/GAE_train.py", line 158, in main
    df = pd.DataFrame(sasp_index, columns=['sasp_index'])
  File "/home/shz19039/miniconda3/envs/climate/lib/python3.10/site-packages/pandas/core/frame.py", line 827, in __init__
    mgr = ndarray_to_mgr(
  File "/home/shz19039/miniconda3/envs/climate/lib/python3.10/site-packages/pandas/core/internals/construction.py", line 336, in ndarray_to_mgr
    _check_values_indices_shape_match(values, index, columns)
  File "/home/shz19039/miniconda3/envs/climate/lib/python3.10/site-packages/pandas/core/internals/construction.py", line 420, in _check_values_indices_shape_match
    raise ValueError(f"Shape of passed values is {passed}, indices imply {implied}")
ValueError: Shape of passed values is (50997, 6), indices imply (50997, 1)
